docs/app/documentation/component-docs/menu/menu-docs.component.html

```diff
@@ -15,3 +15,18 @@ <h2>Menu with Group</h2>
   </p>
 </component-example>
 <code-example [code]="menuGroupHtml" [language]="'HTML'"></code-example>
+
+<separator></separator>
+
+<h2>Menu with keyboard support</h2>
+<description>
+    Menu keyboard support can be added on every component, which is using <code>MenuComponent</code>. It simulates
+    click event on <code>Enter</code> or <code>Space</code> key down. Also it's possible to overwrite functions, which
+    are responsible for situation, when focus escape before <code>focusEscapeBeforeList</code> or after
+    <code>focusEscapeAfterList</code> list. To try it, use arrow up/down.
```

Comment:
It is also possible to override functions for a variety of situations. For example, when the focus should escape the element. Use the up and down arrows to test the component below.